### PR TITLE
Add missing keys

### PR DESCRIPTION
**Pull request by 1pretz1**
Normalises and adds missing locales.

More info in the commits ->

---

Trello:
https://trello.com/c/kYXvexWo/2455-add-missing-keys-to-locale-files-3

### CODE REVIEW

File: config/locales/ar.yml [hunk 1]
```diff
@@ -212,6 +227,18 @@ ar:
         other: مسودة النصوص
         two:
         zero:
+      edition:
```
Comment (brucebolt) [inline config/locales/ar.yml:230]: I think the pluralisation needs updating here, to reflect the plurals used in each language (e.g. Arabic has `one`, `many`, `few`, `other`, `two` and `zero` that will need placeholder keys).

**Review by brucebolt — COMMENTED** [review]
Great work!

I've added a comment about plurals: we need to add the correct keys for each language (i.e. to keep in line with the work done in [this card](https://trello.com/c/lkVB57ri/488-3-standardise-keys-for-plural-forms-across-locale-files)).  Other than that, everything else looks really good.

File: config/locales/ar.yml (hunk 1 above)
Comment (1pretz1) [reply config/locales/ar.yml:230]: Ah yes you're right, good spot! I have added / removed pluralisations for the other apps but Whitehall must've slipped through the net.

**Comment by 1pretz1** [comment]
Thanks for the review @brucebolt, I believe the correct pluralisations per language have now been added. Would you mind giving it another gander?

**Review by brucebolt — APPROVED** [review]
Thanks for making that change.  Everything looks good now 🎉.

There's a test failure, but it looks completely unrelated 😕, maybe a flaky test?

**Comment by 1pretz1** [comment]
Hmm, yeah it does seem to be a flakey test. I reran the tests and they passed. 

I've had a little dig into it and can't see anything obvious to fix and can't reproduce the failure so I'm going to stop there before I end up in a rabbit hole 🐰

**Pull request merged**